aspnet/tutorials/nano-server.rst

```diff
@@ -97,26 +109,34 @@ In the ``system.webServer`` section group, update the handlers section to allow
   :dedent: 8
   :emphasize-lines: 2
 
-Add ``httpPlatformHandler`` to the ``globalModules`` section
+Add ``AspNetCoreModule`` to the ``globalModules`` section
 
 .. literalinclude:: nano-server/applicationHost.config
   :language: xml
   :lines: 210,224
   :dedent: 8
   :emphasize-lines: 2
 
-Additionally, add ``httpPlatformHandler`` to the ``modules`` section
+Additionally, add ``AspNetCoreModule`` to the ``modules`` section
 
 .. literalinclude:: nano-server/applicationHost.config
   :language: xml
   :lines: 275,286
   :dedent: 8
   :emphasize-lines: 2
 
+Installing .NET Core Framework
+------------------------------
+
+If you published a portable app, you require .NET Core to be installed on your target machine. You can execute the following Powershell script in a remote Powershell session to install the .NET Framework to your Nano Server.
```

Comment:
>If you published a portable app, you require .NET Core to be installed on your target machine.
If you published a portable app, .NET Core must be installed on the target machine.